### DISCUSSION TITLE
Lesson 6, Ubuntu installation for WSL

### DISCUSSION THREAD

**Discussion by OMGitsNIK** [Cyfrin/foundry-full-course-cu#466] [Q&A] [answered]
I am getting the following error after downloading, and while installing Ubuntu for my Windows PC.
```
WslRegisterDistribution failed with error: 0x800701bc
Error: 0x800701bc WSL 2 requires an update to its kernel component. For information please visit https://aka.ms/wsl2kernel
```
What should I do to fix this issue ?

**Comment by Niferu** [comment, marked as answer]
Hello @OMGitsNIK 

You just have to enable below in your system:

`Microsoft-Windows-Subsystem-Linux`
`VirtualMachinePlatform`

**Reply by OMGitsNIK** [reply]
Thanks
